TASVideos/Pages/Systems/Index.cshtml.cs

```diff
@@ -17,7 +17,7 @@ public IndexModel(IGameSystemService systemService)
 	public async Task OnGet()
 	{
 		Systems = (await _systemService.GetAll())
-			.OrderBy(s => s.Id)
+			.OrderBy(s => s.DisplayName)
```

Comment:
Will it also make it the first column? May not be important, but probably nicer for the user (sorry I forgot to mention it originally).

Reply:
No it will not. But we can change that as well.

Reply:
Pretty please :) It makes sense when column order reflects sorting priority.

Reply:
Looks good?
![image](https://user-images.githubusercontent.com/54983451/205335667-9a8af4ab-bc9d-43d1-9a78-c373eba25871.png)

Reply:
Perfect!